python/fate_flow/pipelined_model/publish_model.py

```diff
@@ -108,3 +111,43 @@ def download_model(request_data):
     model_data = model.collect_models(in_bytes=True)
     return model_data
 
+
+def convert_homo_model(request_data):
+    model_id = request_data.get("model_id")
+    model_version = request_data.get("model_version")
+    model = pipelined_model.PipelinedModel(model_id=model_id, model_version=model_version)
+    if not model.exists():
+        return 100, 'Model {} {} does not exist'.format(model_id, model_version), None
+
+    with open(model.define_meta_path, "r", encoding="utf-8") as fr:
+        define_index = yaml.safe_load(fr)
+
+    framework_name = request_data.get("framework_name")
+    detail = []
+    for key, value in define_index.get("model_proto", {}).items():
+        if key == 'pipeline':
+            continue
+        for v in value.keys():
+            buffer_obj = model.read_component_model(key, v)
+            module_name = define_index.get("component_define", {}).get(key, {}).get('module_name')
+            converted_framework, converted_model = model_convert(model_contents=buffer_obj,
+                                                                 module_name=module_name,
+                                                                 framework_name=framework_name)
+            if converted_model:
+                converted_model_dir = os.path.join(model.variables_data_path, key, "converted_model")
```

Comment:
The path  missing model_alias

Reply:
I was thinking "model_alias" is "converted_model" - at the same level as "model". So the folder layout looks like

```
variables/data/homo_lr_0/
├── converted_model
│   └── sklearn.joblib
└── model
    ├── HomoLogisticRegressionMeta
    └── HomoLogisticRegressionParam
```

Could you help see if that is ok?

Reply:
variables
├── data
│   └── dataio_0
│       └── dataio
│           ├── DataIOMeta
│           └── DataIOParam

and refer https://github.com/FederatedAI/FATE/blob/6c5263d8d1ed0423884fda10dd30304cd6d25f8d/python/fate_flow/pipelined_model/pipelined_model.py#L61

Reply:
Got it. I moved the folder under the model_alias folder now.